nixos/tests/k3s/multi-node.nix

```diff
@@ -142,11 +142,9 @@ import ../make-test-python.nix ({ pkgs, lib, ... }:
       server.wait_until_succeeds("k3s kubectl get node agent")
 
       for m in machines:
-      '' # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
-      + lib.optionalString (!pkgs.stdenv.isAarch64) ''m.succeed("k3s check-config")'' + ''
-          m.succeed(
-              "${pauseImage} | k3s ctr image import -"
-          )
+          ${ # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
```

Comment:
I think this reads better if we add a python variable instead.

See https://github.com/euank/nixpkgs/commit/9d477c37ac1e05195d192a4b746f9c3764816930

I wrote up that commit before seeing that you already had a PR; please feel free to take those changes or modify what you've got here as you see fit!

Reply:
We can use your PR. (That is ready, right?)

Reply:
Ah, yup, happy to open it as a PR. Doing so now

Reply:
@euank A tangent issue, that I'd like your input on, if you don't mind.
Recently metrics-server broke because of port change to 10250 and needs flag `--kubelet-insecure-tls`. But then, whenever I edit/apply the change to metrics-server's deployment resource, the change gets rolled back. I don't know why it rolls back, no message, or anything, any clue?

Reply:
Opened the replacement PR over here: https://github.com/NixOS/nixpkgs/pull/198390

Regarding the metrics server stuff: k3s has some pre-packages component it automatically applies. You can disable the metrics-server one with the server flag `--disable metrics-server`.

If you don't pass that flag, it writes em to `/var/lib/rancher/k3s/server/manifests/metrics-server`.

There's then a second layer, which is the addon controller that applies em:

```
$ sudo k3s kubectl get addon -n kube-system metrics-server-deployment -o yaml
...
source: /var/lib/rancher/k3s/server/manifests/metrics-server/metrics-server-deployment.yaml
...
```

Which is periodically then applied every 15s via this:

https://github.com/k3s-io/k3s/blob/76729d813b434c3573a67ad5b18429347c13803b/pkg/deploy/controller.go#L82-L98

... It's something like that I think anyway. Deleting the addon, those files, and adding the disable flag should let you manage it as a normal deployment if that's easier for you.

Reply:
Thanks! I am always glad we have you here. :-)

Reply:
You're too kind, thank you.
I really appreciate having you around too!